crates/chainspec/src/spec.rs

```diff
@@ -39,6 +39,9 @@ use reth_network_peers::{
     op_testnet_nodes, sepolia_nodes,
 };
 
+#[cfg(feature = "optimism")]
+use op_alloy_rpc_types::genesis::OptimismChainInfo;
```

Comment:
please use full path instead of feature gating an import

Reply:
Fixed. Thx